adapters/bidder.go

```diff
@@ -137,6 +137,7 @@ func (r *RequestData) SetBasicAuth(username string, password string) {
 
 type ExtraRequestInfo struct {
 	PbsEntryPoint metrics.RequestType
+	SecGPCFlag    string
```

Comment:
I'm not sure we need to use the exact cookie name as the variable. IMHO, a more straight forward name, such as "GlobalPrivacyControl" or "GlobalPrivacyControlHeader" would be easier to read in adapter code.

Reply:
Thanks for spending your time reviewing this draft PR. I am new to the community and I hope to contribute regularly. I have updated this with your recommendation. Thanks!

Reply:
Welcome to the community. We're very welcoming of new regular contributors. Please let us know via a comment on the GitHub Issues what you'd like to work on next so there's no duplicate effort.